### PR TITLE
Add description for mac/mtu/promisc in tuning README.md

### LINKED ISSUE #199: Missing MAC/promiscas config documents in tuning plugin.
I will submit PR soon....

### PR DESCRIPTION
**Pull request by s1061123**
This diff adds documents for #177 change (mac/mtu/promisc) in tuning README.md. Fixes #199.

### CODE REVIEW

File: plugins/meta/tuning/README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 ## Overview
 
-This plugin can change some system controls (sysctls) in the network namespace.
+This plugin can change some system controls (sysctls) and several netlink attributes (promiscuous mode and mac) in the network namespace.
```
Comment (dcbw) [inline plugins/meta/tuning/README.md]: I would change this to "several interface attributes (promiscuous mode, MTU, and MAC address)"

File: plugins/meta/tuning/README.md [hunk 2]
```diff
@@ -32,3 +32,32 @@ Some network sysctls are documented in the Linux sources:
 - [Documentation/sysctl/net.txt](https://www.kernel.org/doc/Documentation/sysctl/net.txt)
 - [Documentation/networking/ip-sysctl.txt](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt)
 - [Documentation/networking/](https://www.kernel.org/doc/Documentation/networking/)
+
+## Netlink Operation
```
Comment (dcbw) [inline plugins/meta/tuning/README.md]: I would change to "Interface Attribute Operation"

File: plugins/meta/tuning/README.md [hunk 3]
```diff
@@ -32,3 +32,32 @@ Some network sysctls are documented in the Linux sources:
...
+{
+  "name": "mytuning",
+  "type": "tuning",
+  "promisc": true,
+  "mac": "c2:b0:57:49:47:f1",
+  "mtu": 1454
+}
+```
+
+## Netlink configuration reference
```
Comment (dcbw) [inline plugins/meta/tuning/README.md]: And "Interface attribute configuration reference"

File: plugins/meta/tuning/README.md (hunk 1 above)
Comment (s1061123) [reply plugins/meta/tuning/README.md]: Thanks. Changed.

File: plugins/meta/tuning/README.md (hunk 2 above)
Comment (s1061123) [reply plugins/meta/tuning/README.md]: Changed.

File: plugins/meta/tuning/README.md (hunk 3 above)
Comment (s1061123) [reply plugins/meta/tuning/README.md]: Fixed.

File: plugins/meta/tuning/README.md [hunk 4]
```diff
@@ -32,3 +32,32 @@ Some network sysctls are documented in the Linux sources:
...
+  "mac": "c2:b0:57:49:47:f1",
+  "mtu": 1454
+}
+```
+
+## Interface attribute configuration reference
+
+* `mac` (string, optional): MAC address (i.e. hardware address) of interface
+* `mtu` (integer, optional): MTU of interface
+* `promisc` (bool, optional): Change the promiscas mode of interface
```
Comment (jellonek) [inline plugins/meta/tuning/README.md]: Shouldn't that be `promiscuous`? Looks like a typo.

Comment (s1061123) [reply plugins/meta/tuning/README.md]: Good catch! thank you so much!

Comment (s1061123) [reply plugins/meta/tuning/README.md]: Also fixed same typo in plugins/meta/tuning/tuning_test.go.

Comment (jellonek) [reply plugins/meta/tuning/README.md]: I see this fix tuning_test.go, but what with this one?

Comment (s1061123) [reply plugins/meta/tuning/README.md]: Uploaded.

**Comment by jellonek** [comment]
@dcbw PTAL

**Comment by dcbw** [comment]
LGTM

**Pull request merged**